test/lint/golangci.sh

```diff
@@ -28,7 +28,8 @@ fi
 
 # Fetch the reference if it doesn't exist (Github uses a shallow clone).
 if ! git show-ref --quiet "refs/heads/${target_branch}" >/dev/null 2>&1; then
-    git fetch origin "${target_branch}"
+    echo "Convert shallow clone into treeless clone"
+    git fetch --filter=tree:0 origin "${target_branch}"
```

Comment:
what is the difference here?

Reply:
I read the linked article and updated the PR description with the answer to my question :)